src/resolutions/lesson-02/5-avoid-return-type.ts

```diff
@@ -0,0 +1,18 @@
+import { Equal, Expect } from "../../../helper";
+import { expect, it} from "vitest";
+
+// 5. avoid an any return type in a fetch calls using generics
+
+const fetchData = async <T>(url: string): Promise<T> => {
+  const data = await fetch(url).then((response) => response.json());
+  return data;
+};
+
```

Comment:
1. con solo anotar el tipo de retorno de nuestra función, si los test pasan pero nuestra variable data dentro de nuestro cuerpo de la función es de tipo `:any` por lo que podemos hacer cambiar su valor. 
la intención de este ejercicio era evitar el `:any` a nivel de nuestra variable data.

Reply:
Thx, addressed in next commit.